.github/workflows/releaseinfo.yml

```diff
@@ -0,0 +1,43 @@
+name: write release information to file
+
+on: workflow_dispatch
+
+jobs:
+  release_info:
+    runs-on: ubuntu-latest
+    steps:
+      - id: info
+        uses: konveyor/get-release-info@v1
+        with:
+          owner: konveyor
+          repo: move2kube
+          token: ${{ secrets.GITHUB_TOKEN }}
+      - uses: actions/checkout@v2
+      - run: echo '${{ steps.info.outputs.release_info }}' > _data/releaseinfo.json
+      - id: get_asset_url
+        uses: actions/github-script@v3
+        with:
+          github-token: ${{ secrets.MOVE2KUBE_PATOKEN }}
+          script: |
+            const response = await github.repos.listReleases({ owner:context.repo.owner, repo:'move2kube-operator' });
```

Comment:
Not for this PR, but we should move our helm chart to move2kube-ui repo

Reply:
ok